python/eos/tasks.py

```diff
@@ -507,17 +508,17 @@ def corner_plot(analysis_file:str, posterior:str, base_directory:str='./'):
             ax = axes[i, j]
             xsamples = f.samples[:, j]
             ysamples = f.samples[:, i]
-            xmin = np.min(xsamples)
-            xmax = np.max(xsamples)
-            ymin = np.min(ysamples)
-            ymax = np.max(ysamples)
+            xmin = _np.min(xsamples)
+            xmax = _np.max(xsamples)
+            ymin = _np.min(ysamples)
+            ymax = _np.max(ysamples)
+            #ax.set_aspect(_np.diff((xmin, xmax))[0] / _np.diff((ymin, ymax))[0])
+            ax.hist2d(xsamples, ysamples, weights=f.weights, alpha=1.0, bins=100, cmap='Greys')
             ax.set_xlim((xmin, xmax))
             ax.set_ylim((ymin, ymax))
-            ax.set_aspect(np.diff((xmin, xmax))[0] / np.diff((ymin, ymax))[0])
-            ax.hist2d(xsamples, ysamples, weights=weights, alpha=1.0, bins=100, cmap='Greys')
 
     fig.tight_layout()
-    fig.savefig(os.path.join(base_directory, posterior, 'plots', 'corner-plot.pdf'))
+    fig.savefig(os.path.join(base_directory, posterior, 'plots', 'corner-plot.png'))
```

Comment:
I think we discussed that this should be an optional keyword for the task, e.g., ``format='pdf'``.